components/assistant-ui/calculator-tool-ui.tsx

```
@@ -239,7 +224,10 @@ export const CalculatorToolUI: ToolCallContentPartComponent = ({
     if (!args) return null;
 
     const resolvedResult = result ?? output;
-    const normalizedResult = normalizeCalculatorResult(resolvedResult);
+    const normalizedResult = useMemo(
```

Biome (2.4.4)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)